lib/Optimizer/IROptimizer.cpp

```diff
@@ -40,6 +40,13 @@ static llvm::cl::opt<bool>
     instrumentDebug("instrument-debug",
                     llvm::cl::desc("Instrument the IR for debugging"),
                     llvm::cl::init(false), llvm::cl::Hidden);
+
+static llvm::cl::list<std::string> instrumentDebugOnly(
```

Comment:
can you specify in the comment how this interacts with the "instrument-debug" option above.
What happens if instrument-debug == false and instrumentDebugOnly has something.